mmedit/datasets/pipelines/sr_resize.py

```diff
@@ -81,9 +151,11 @@ def __call__(self, results):
 
     def __repr__(self):
         repr_str = self.__class__.__name__
-        repr_str += (f'scale_min={self.scale_min}, '
+        repr_str += (f' scale_min={self.scale_min}, '
```

Comment:
Remove extra space

Reply:
I see, you can use the f-str format

```python
repr_str = (f'{self.__class__.__name__} ({scale_min={self.scale_min}, '
                  f'scale_max={self.scale_max}, '
                  ...)
```

Reply:
It is consistent with the commonly used description in the repo and can be revised in the future.